src/tribler/gui/widgets/channelcontentswidget.py

```diff
@@ -133,14 +131,19 @@ def on_channel_committed(self, response):
             self.update_labels()
 
     def commit_channels(self, checked=False):  # pylint: disable=W0613
-        TriblerNetworkRequest("channels/mychannel/0/commit", self.on_channel_committed, method='POST')
+        request = Request(
+            endpoint="channels/mychannel/0/commit",
+            on_finish=self.on_channel_committed,
+            method=Request.POST
+        )
+        request_manager.add(request)
```

Comment:
The explicit creation of `Request` object looks a bit verbose, and (as I showed for `FileDownloadRequest`) it can be error-prone. Adding simple helper methods to `RequestManager` can make the code more concise and less error-prone:

```python
request_manager.post("channels/mychannel/0/commit", on_finish=self.on_channel_committed)
```